### ISSUE TITLE
[Accessibility] vscode doesn't play sound when cursor is at error line

### ISSUE THREAD

**Issue by mo29cg** [microsoft/vscode#166074]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.7.2
- OS Version:  windows

Steps to Reproduce:

1. create a file with error
2. turn on `cue: line has an error" settings
3. move the cursor to there

**Comment by VSCodeTriageBot** [comment]
Thanks for creating this issue! It looks like you may be using an old version of VS Code, the latest stable release is 1.73.1. Please try upgrading to the latest version and checking whether this issue remains.

Happy Coding!

**Comment by mo29cg** [comment]
it still occurs with 1.73.1

**Comment by meganrogge** [comment]
yeah I can reproduce this issue. @isidorn  is this expected?

**Comment by isidorn** [comment]
This is not expected. Sounds should be played when cursor is navigated to the error line.
@hediet worked on this initially and might have more details

**Comment by jeanbrazcosta** [comment]
I also have the same problem, but this only occurs in the insider version, I have the latest version and no sound is played
Issue:

167679

**Comment by meganrogge** [comment]
They play in OSS, but not insider's. @hediet any idea why that would be?

**Comment by isidorn** [comment]
@meganrogge if that is the case then it is probably related to bundling. Does that sound play when you F1 > list audio cues?

**Comment by meganrogge** [comment]
It does play when I list audio cues in both OSS and insider's.

**Comment by meganrogge** [comment]
It also plays for terminal quick fixes - suggesting there's a bug in `audioCueLineFeatureContribution.ts` because I can also repro with the other line feature types

**Comment by meganrogge** [comment]
This has been working for me lately, though it was definitely broken at one pt.

**Comment by jrieken** [comment]
* I have `audioCues.lineHasError` set to `on`
* I navigate to a line with an error-squiggle but there is no sound being played
